### DISCUSSION TITLE
Allow to Hide/Reorder/Pre-Select Ship/Pickup Selection as well using functions

### DISCUSSION THREAD

**Discussion by SaeedYasin** [Shopify/function-examples#240] [API Ideas]
When a product is pickup only, then showing "Ship" option by default makes no sense and confuses customer. Allow to hide it conditionally or reorder it to make pickup default option for that order.

<img width="620" alt="Screenshot 2023-05-15 at 1 04 15 PM" src="https://github.com/Shopify/function-examples/assets/26478201/f2ae6e12-99b2-454f-8b21-a9e1d07b9eb4">

**Comment by blanklob** [comment]
Any update on this guys?
